docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md

```diff
@@ -17,295 +17,184 @@ related:
+use Spryker\Zed\CmsBlockCategoryConnector\Communication\Plugin\CategoryGui\CmsBlockCategoryRelationReadPlugin;
+use Spryker\Zed\CmsBlockCategoryConnector\Communication\Plugin\CategoryGui\CmsBlockSubformCategoryFormPlugin;
 
-class CategoryDependencyProvider extends SprykerDependencyProvider
+/**
+ * @method \Spryker\Zed\CategoryGui\CategoryGuiConfig getConfig()
+ */
+class CategoryGuiDependencyProvider extends SpykerCategoryGuiDependencyProvider
 {
-	...
-	/**
-     * @return \Spryker\Zed\Category\Dependency\Plugin\CategoryRelationUpdatePluginInterface[]
+    /**
+     * @return array<\Spryker\Zed\CategoryGuiExtension\Dependency\Plugin\CategoryFormPluginInterface>
      */
-    protected function getRelationUpdatePluginStack()
+    protected function getCategoryFormPlugins(): array
     {
-        return array_merge(
-            [
-				...
-                new CategoryFormPlugin(),
-            ],
-            parent::getRelationUpdatePluginStack()
-        );
+        return [
+            new CmsBlockSubformCategoryFormPlugin(),
+        ];
     }
 
-	/**
-     * @return array
+    /**
+     * @return array<\Spryker\Zed\CategoryGuiExtension\Dependency\Plugin\CategoryRelationReadPluginInterface>
      */
-    protected function getCategoryFormPlugins()
+    protected function getCategoryRelationReadPlugins(): array
     {
-        return array_merge(parent::getCategoryFormPlugins(), [
-			...
-            new CategoryFormPlugin()
-        ]);
+        return [
+            new CmsBlockCategoryRelationReadPlugin(),
+        ];
     }
 }
 ```
 
-13. Optional: To show which categories are assigned to a block on a block view page, register the category list plugin by adding `CmsBlockCategoryListViewPlugin` to the CMS Block GUI dependency provider:
+6. Optional: To show which categories are assigned to a block on a block view page, register the category list plugin by adding `CmsBlockCategoryListViewPlugin` to the CMS Block GUI dependency provider:
 
 ```php
 <?php
+
 namespace Pyz\Zed\CmsBlockGui;
 
-...
+use Spryker\Zed\CmsBlockCategoryConnector\Communication\Plugin\CmsBlockCategoryListViewPlugin;
+use Spryker\Zed\CmsBlockGui\CmsBlockGuiDependencyProvider as SprykerCmsBlockGuiDependencyProvider;
 
-class CmsBlockGuiDependencyProvider extends CmsBlockGuiCmsBlockGuiDependencyProvider
+class CmsBlockGuiDependencyProvider extends SprykerCmsBlockGuiDependencyProvider
 {
-	...
-
-	/**
-	* @return array
-	*/
-	protected function getCmsBlockViewPlugins()
-	{
-		return array_merge(parent::getCmsBlockViewPlugins(), [
-			new CmsBlockCategoryListViewPlugin(),
-		]);
-	}
+    /**
+     * @return array<\Spryker\Zed\CmsBlockGui\Communication\Plugin\CmsBlockViewPluginInterface>
+     */
+    protected function getCmsBlockViewPlugins(): array
+    {
+        return array_merge(parent::getCmsBlockViewPlugins(), [
+            new CmsBlockCategoryListViewPlugin(),
+        ]);
+    }
 }
 ```
 
 **To configure the block:**
```

Comment:
Do I understand correctly that this is the procedure of adding a CMS block to the category pages?

Reply:
Yes